### PR TITLE
formatted remote data manager example

### LINKED ISSUE #3612: Formatting of remote data manager example
The [remote data manager example](https://docs.sunpy.org/en/latest/generated/gallery/remote_data_manager.html#sphx-glr-generated-gallery-remote-data-manager-py) is listed on its own instead of in one of the other example gallery categories (e.g. "Map", "Acquiring Data"). Is this intentional?

Additionally, the one line description just below the header is formatted a bit strange (i.e. there's some bold type in there and extra line breaks) compared to the other examples.

### PR DESCRIPTION
**Pull request by iamrajiv**
### Description

Formatted `remote_data_manager.py`  which fixes bold type and extra line breaks

Fixes #3612

### CODE REVIEW

File: changelog/3615.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fixed bugs in `remote_data_manager.py` which fixes bold type and extra line
+breaks in remote data manager.
```
Comment (nabobalis) [inline changelog/3615.bugfix.rst]: This is not a bug fix but a documentation change, could you reword the changelog and the type of the change please?

**Comment by nabobalis** [comment]
Thanks @iamrajiv

**Comment by iamrajiv** [comment]
> Thanks @iamrajiv

thank you for guidance @nabobalis :smiley:

**Comment by nabobalis** [comment]
Online server failed.

**Pull request merged**